airbyte-cdk/python/.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.73.0
+current_version = 0.73.1
```

Comment:
no need to bump the version. this is done automatically by the publish github action

Reply:
Fixed